lib/src/models/spending_data_source.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'spending_data_source.freezed.dart';
+
+@freezed
+sealed class SpendingDataSource with _$SpendingDataSource {
+  const SpendingDataSource._();
+  const factory SpendingDataSource.none() = SpendingDataSourceNone;
+  const factory SpendingDataSource.path({
+    required List<int> path,
+  }) = SpendingDataSourcePath;
+  const factory SpendingDataSource.scriptHash({
+    required String scriptHashHex,
+  }) = SpendingDataSourceScriptHash;
```

Comment:
An empty line in between constructors / factories / methods makes everything a lot more readable.

Reply:
done